docs/source/geometry.solvers.rst

```diff
@@ -0,0 +1,45 @@
+kornia.geometry.solvers
+=======================
+
+.. currentmodule:: kornia.geometry.solvers
+
+Module containing various geometrical solvers/optimizers.
+
+Polynomial Solvers
+------------------
+
+.. autofunction:: solve_quadratic
+
+Finds the real roots of a quadratic equation
+
+.. math:: coeffs[0]x^2 + coeffs[1]x + coeffs[2] = 0
```

Comment:
i think that all this could be just included in the function docstring, including the small snippet (which tested too)

Reply:
+1

Reply:
what do you think about that note?  need to remove that from solvers.rst?